src/packages/@ncigdc/modern_components/ClinicalVariableCard/ClinicalVariableCard.js

```diff
@@ -144,7 +163,49 @@ const enhance = compose(
       viewer && viewer.explore.cases.facets
         ? tryParseJSON(viewer.explore.cases.facets, {})
         : {},
-  }))
+  })),
+  withState('selectedSurvivalData', 'setSelectedSurvivalData', {}),
+  withState('overallSurvivalData', 'setOverallSurvivalData', {}),
+  withState('selectedSurvivalLoadingId', 'setSelectedSurvivalLoadingId', ''),
+  withState('state', 'setState', initialState),
```

Comment:
same as above, is it possible to change this to a specific value and not use generic state?

Reply:
added a specific state field